### PR TITLE
feat: remove integration name from sensor names

### LINKED ISSUE #49: Remove integration name from entity name
I'm thinking we want to skip the integration name as part of the entity name. Looking at other integrations, it seems like you don't usually include that. It's still clear from entity id and integration name where it comes from. If it's desirable, we could put it in the config flow - "Prefix entity names with integration name" as a checkbox or possibly a text field. But I'm not sure this is needed? The entity id still shows which integration is't part of, should you have multiple Energy Hubs configured.

image

So for example we should just use `Consumption Power`instead of `Ferroamp Consumption Power` as default name of the entities.

### PR DESCRIPTION
**Pull request by argoyle**
Closes #49

### CODE REVIEW

**Review by henricm — CHANGES_REQUESTED** [review]
As discussed in #

**Comment by henricm** [comment]
I've made a branch which illustrates my thoughts. Seems to work in my installation:
https://github.com/henricm/ha-ferroamp/tree/remove-integration-name-from-entity-name

Please let me know what you think. This results in removing integration name from display name only:
image

image

**Comment by henricm** [comment]
And now I also realise this is exactly what you wrote, @argoyle, in [your comment](https://github.com/henricm/ha-ferroamp/issues/49#issuecomment-891271391) on the issue - I guess I needed to try it out to understand :-)

File: custom_components/ferroamp/sensor.py [hunk 1]
```diff
@@ -531,7 +559,7 @@ async def options_update_listener(hass, entry):
 class FerroampSensor(SensorEntity, RestoreEntity):
     """Representation of a Ferroamp Sensor."""
 
-    def __init__(self, name, unit, icon, device_id, device_name, interval, config_id, **kwargs):
+    def __init__(self, name, slug, unit, icon, device_id, device_name, interval, config_id, **kwargs):
```
Comment (henricm) [inline custom_components/ferroamp/sensor.py:562]: Maybe change `slug` to `entity_prefix` or send integration name instead and slugify everything at once when setting `entity_id`? I got a bit confused when reading the code as it is now 🙂

**Review by henricm — APPROVED** [review]
I think it looks great. Just minor naming feedback that you could consider if you want.

File: custom_components/ferroamp/sensor.py (hunk 1 above)
Comment (argoyle) [reply custom_components/ferroamp/sensor.py:562]: Good point! I've looked at it myself as well. Changing.

**Pull request merged**